node/pkg/governor/governor_monitoring.go

```diff
@@ -213,19 +213,19 @@ func (gov *ChainGovernor) ReleasePendingVAA(vaaId string) (string, error) {
 }
 
 // Admin command to reset the release timer for a pending VAA, extending it to the configured limit.
-func (gov *ChainGovernor) ResetReleaseTimer(vaaId string) (string, error) {
-	return gov.resetReleaseTimerForTime(vaaId, time.Now())
+func (gov *ChainGovernor) ResetReleaseTimer(vaaId string, numDays uint32) (string, error) {
+	return gov.resetReleaseTimerForTime(vaaId, time.Now(), numDays)
 }
 
-func (gov *ChainGovernor) resetReleaseTimerForTime(vaaId string, now time.Time) (string, error) {
+func (gov *ChainGovernor) resetReleaseTimerForTime(vaaId string, now time.Time, numDays uint32) (string, error) {
 	gov.mutex.Lock()
 	defer gov.mutex.Unlock()
 
 	for _, ce := range gov.chains {
 		for _, pe := range ce.pending {
 			msgId := pe.dbData.Msg.MessageIDString()
 			if msgId == vaaId {
-				pe.dbData.ReleaseTime = now.Add(maxEnqueuedTime)
+				pe.dbData.ReleaseTime = now.Add(maxEnqueuedTime * time.Duration(numDays))
```

Comment:
`maxEnqueuedTime` is already a duration, so I don't think we need to wrap `numDays` as well?

Reply:
Can't multiply time.Duration with a uint32, hence the wrap. That being said, it doesn't really make sense to have `maxEnqueuedTime` since we are directly specifying the enqueue time. So going to update this a bit.